docs/layouts/shortcodes/generated/metric_reporters_section_zh.html

```diff
@@ -0,0 +1,66 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">键</th>
+            <th class="text-left" style="width: 15%">默认值</th>
+            <th class="text-left" style="width: 10%">数据类型</th>
+            <th class="text-left" style="width: 55%">描述</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>metrics.reporter.&lt;name&gt;.class</h5></td>
```

Comment:
This item seems removed from the English version?

Reply:
Fixed!